finance_dl/scrape_lib.py

```diff
@@ -143,6 +143,7 @@ def __init__(self, download_dir=None, connect=None, chromedriver_bin='finance-dl
 
         self.chromedriver_bin = chromedriver_bin
         chrome_options = webdriver.ChromeOptions()
+        chrome_options.binary_location = os.getenv("CHROMEDRIVER_CHROME_BINARY")
```

Comment:
Needs a default value for those who don't set an environment variable.

Reply:
The environment variable name was invented for this patch, it's not used by anything other than this line, so yes, it's platform-independent (well, I guess it won't work on a system that doesn't support 26-character environment variable names with underscores in them, but I don't think there are such systems that also run Chrome).

We actually don't need an explicit default; None is handled properly by Selenium (it results in not passing a `binary` option to Chromedriver, which in turn results in Chromedriver using its default behavior of attempting to locate the system Chrome, as happens without this change).